cmd/incus-migrate/main_migrate.go

```diff
@@ -659,8 +840,10 @@ func (c *cmdMigrate) run(_ *cobra.Command, _ []string) error {
 What would you like to create?
 1) Container
 2) Virtual Machine
+3) Custom Volume (from filesystem)
+4) Custom Volume (from disk)
```

Comment:
@presztak I'll merge the PR because the logic is fine, but in a follow-up we should be able to turn that just into a single `Custom volume` option and detect whether it's a filesystem or a disk based on what the user provides us.

If it's a directory, it's a filesystem, if not, it's a disk.